core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -348,3 +348,19 @@ private[spark] class SparkSQLFeatureNotSupportedException(
 
   override def getErrorClass: String = errorClass
 }
+
+/**
+ * Exception thrown from Spark Streaming framework.
+ */
+private[spark] class SparkStreamingException(
```

Comment:
Could you re-use the existing exception: `SparkRuntimeException`, please. We don't need to introduce additional things as we already have error classes that users can use to distinguish errors.